pages/en/developers/sandbox-node.mdx

```diff
@@ -24,7 +24,8 @@ docker run \
   -d \
   --name mina \
   -e RUN_DEMO=true \
-  minaprotocol/mina-daemon:1.2.0-fe51f1e-devnet
+  -e CODA_PRIVKEY_PASS='' \
+  minaprotocol/mina-daemon:1.2.0-fe51f1e-demo
```

Comment:
i wanna say that the latest version is 1.2.2, `minaprotocol/mina-daemon:1.2.2-feee67c-devnet`.  And do we even have a `-demo` container? why can't we just use the normal container?  would overall be better if we collapse these functionalities so we don't have multiple slightly different containers floating around.

Reply:
@lk86 also agrees on the container image - I have tested ad command and updated the code for this change as well.